File: PigFarm/Models/Room.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace PigFarm.Models
{
    public partial class Room
    {
        public decimal Id { get; set; }
        public string FarmGuid { get; set; }
        public string AreaGuid { get; set; }
        public string BarnGuid { get; set; }
        public string Type { get; set; }
        public string RoomNo { get; set; }
        public string RoomName { get; set; }
        public string RoomPrincipal { get; set; }
        public decimal? RoomLength { get; set; }
        public decimal? RoomWidth { get; set; }
        public string Comment { get; set; }
        public DateTime? CreateDate { get; set; }
        public decimal? CreateBy { get; set; }
        public DateTime? UpdateDate { get; set; }
        public decimal? UpdateBy { get; set; }
        public DateTime? DeleteDate { get; set; }
        public decimal? DeleteBy { get; set; }
        public decimal? Status { get; set; }
        public string Guid { get; set; }
        public string CancelFlag { get; set; }
        public DateTime? StartDate { get; set; }
        public DateTime? EndDate { get; set; }
    }
}

File: App2/src/app/_core/_model/farms/room.ts
export interface Room {
  id: number;
  farmGuid: string;
  areaGuid: string;
  barnGuid: string;
  type: string;
  roomNo: string;
  roomName: string;
  roomPrincipal: string;
  roomLength: number | null;
  roomWidth: number | null;
  comment: string;
  createDate: string | null;
  createBy: number | null;
  updateDate: string | null;
  updateBy: number | null;
  deleteDate: string | null;
  deleteBy: number | null;
  status: number | null;
  guid: string;
  cancelFlag: string;
  startDate: any | null;
  endDate: any | null;
}
export interface RoomScreen {
  id: number;
  roomName: string;
  guid: string;
}
